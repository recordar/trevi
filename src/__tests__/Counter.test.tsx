// import * as React from 'react';
// import { render } from 'enzyme';
// import Counter, { ICounterProps } from '../component/Counter';

// const testProps: ICounterProps = {
//   number: 1,
//   color: 'red',
//   onIncrement: jest.fn,
//   onDecrement: jest.fn,
//   onSetColor: jest.fn
// }

// const wrapper = render(<Counter {...testProps} />);

// describe('<Counter>', () => {
//   it('스냅샷 테스트', () => {
//     expect(wrapper).toMatchSnapshot();
//   });
// });
