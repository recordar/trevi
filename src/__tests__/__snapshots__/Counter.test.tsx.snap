// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Counter> 스냅샷 테스트 1`] = `
<div
  class="Counter"
  style="background-color:red"
>
  1
</div>
`;
